src/ops.cpp

```diff
@@ -84,7 +84,7 @@ const std::unordered_map<PrimOp, std::string> primop2str(
       { Rotate_Left, "rotate_left" },
       { Rotate_Right, "rotate_right" },
       { BV_To_Nat, "bv2nat" },
-      { Int_To_BV, "nat2bv" },
+      { Int_To_BV, "int2bv" },
```

Comment:
Is it actually `int2bv` in SMT-LIB? I thought `nat2bv` was right.

Reply:
Well we are testing the generic solver with CVC4, which expects `int2bv`...

Reply:
Gotcha, that's a little strange since it's `nat2bv` here: http://smtlib.cs.uiowa.edu/theories-FixedSizeBitVectors.shtml. Doesn't seem like there's an easy workaround for now. But maybe just something to keep in mind in case it becomes a problem later.

Reply:
well -- in the link that you sent this operator is not defined in the theory itself, but as a helper to define the semantics. So I am not sure it is officially part of the theory.

In partice, i think this can only occur in logics that combine integer arithmetic (linear or not linear) with BV. There are only two such logics in SMT-LIB, and non of their benchmarks include this.

```
SMT-LIB/non-incremental$ ls  |grep BV |grep I
AUFBVDTLIA
AUFBVDTNIA
SMT-LIB/non-incremental$ grep nat2bv -rI AUFBVDTLIA AUFBVDTNIA
SMT-LIB/non-incremental$
```

Maybe this means we should ban this when using a generic solver?

Reply:
Ahh I see, my bad. I think if it's not standardized it's fair to just pick one and go with it. In this case, CVC4 makes the most sense so this is perfect.